src/resolvers/Query/users.ts

```diff
@@ -31,9 +29,7 @@ export const users: QueryResolvers["users"] = async (_parent, args) => {
 
   if (!users[0]) {
     throw new errors.NotFoundError(
-      IN_PRODUCTION !== true
-        ? USER_NOT_FOUND
-        : requestContext.translate(USER_NOT_FOUND_MESSAGE),
```

Comment:
I guess the decrement of 3 hits is due to deletion of these 3 lines, which are now moot.

Reply:
> I guess the decrement of 3 hits is due to deletion of these 3 lines, which are now moot.

But that will not decrease the coverage. It would just decrease the total number of lines.

Reply:
I guess, your PR is correct, as the coverage for users.ts is still at 100%

Reply:
In the CodeCov report, the coverage hasn't changed (it is still `92.8%`). Can't figure out where the difference of `0.01%` is originating from. It you have any idea, do let me know.